modules/osd-persistent-storage-csi-efs-sts.adoc

```diff
@@ -21,11 +21,11 @@ If you perform this procedure after installing the driver and creating volumes,
 
 * You have access to the cluster as a user with the cluster-admin role.
 * AWS account credentials
-* You have installed the AWS EFS CSI Operator.
+* You have installed the Amazon EFS CSI Operator.
 
 .Procedure
 
-To configure the AWS EFS CSI Driver Operator with STS:
+To configure the Amazon EFS CSI Driver Operator with STS:
```

Comment:
Please don't update resource name in the configuration below.

Reply:
I have revoked the resource name in the configuration below. I have updated the instance for Line 28 as it is not command/code/configuration.